stuff/cron/client_coder_month.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/python3
+
+'''Processing coder of month messages.'''
+
+import argparse
+import logging
+import os
+import sys
+import MySQLdb
+import MySQLdb.cursors
+import pika
+
+sys.path.insert(
+    0,
+    os.path.join(
+        os.path.dirname(os.path.dirname(os.path.realpath(__file__))), "."))
+import lib.db   # pylint: disable=wrong-import-position
+import lib.logs  # pylint: disable=wrong-import-position
+
+
+def receive_coder_month_messages(
+        cur: MySQLdb.cursors.BaseCursor,
+        # pylint: disable=unused-argument,
+        rabbit_user: str,
+        rabbit_password: str) -> None:
+    '''Receive coder of month messages'''
+
+    credentials = pika.PlainCredentials(rabbit_user, rabbit_password)
+    parameters = pika.ConnectionParameters('rabbitmq', 5672, '/', credentials)
+    connection = pika.BlockingConnection(parameters)
+    channel = connection.channel()
+    channel.exchange_declare(exchange='logs_exchange', exchange_type='direct')
+    result = channel.queue_declare(queue='', exclusive=True)
+    queue_name = result.method.queue
+    assert queue_name is not None
+    channel.queue_bind(
+        exchange='logs_exchange',
+        queue=queue_name,
+        routing_key="CoderMonthQueue")
+    print('[*] waiting for the messages')
```

Comment:
```suggestion
    logging.info('waiting for the messages')
```

se prefiere usar [`logging`](https://docs.python.org/3/library/logging.html) para mostrar logs. estos se pueden configurar mucho mejor (filtrar por nivel, etc.), y con eso no es necesario agregarle un prefijo! igual con todos los demás `print()`.